src/main/scala/devices/debug/DebugTransport.scala

```diff
@@ -63,63 +64,62 @@ class DTMInfo extends Bundle {
 
 /** A wrapper around JTAG providing a reset signal and manufacturer id. */
 class SystemJTAGIO extends Bundle {
-  val jtag = new JTAGIO(hasTRSTn = false).flip
-  val reset = Bool(INPUT)
-  val mfr_id = UInt(INPUT, 11)
-  val part_number = UInt(INPUT, 16)
-  val version = UInt(INPUT, 4)
+  val jtag = Flipped(new JTAGIO(hasTRSTn = false))
+  val reset = Input(Bool())
+  val mfr_id = Input(UInt(11.W))
+  val part_number = Input(UInt(16.W))
+  val version = Input(UInt(4.W))
 }
 
 class DebugTransportModuleJTAG(debugAddrBits: Int, c: JtagDTMConfig)
   (implicit val p: Parameters) extends Module  {
 
-  val io = new Bundle {
+  val io = IO(new Bundle {
     val dmi = new DMIIO()(p)
     val jtag = Flipped(new JTAGIO(hasTRSTn = false)) // TODO: re-use SystemJTAGIO here?
-    val jtag_reset = Bool(INPUT)
-    val jtag_mfr_id = UInt(INPUT, 11)
-    val jtag_part_number = UInt(INPUT, 16)
-    val jtag_version = UInt(INPUT, 4)
-    val fsmReset = Bool(OUTPUT)
-  }
+    val jtag_reset = Input(Bool())
+    val jtag_mfr_id = Input(UInt(11.W))
+    val jtag_part_number = Input(UInt(16.W))
+    val jtag_version = Input(UInt(4.W))
+    val fsmReset = Output(Bool())
+  })
 
   //--------------------------------------------------------
   // Reg and Wire Declarations
 
   val dtmInfo = Wire(new DTMInfo)
 
-  val busyReg = RegInit(Bool(false))
-  val stickyBusyReg = RegInit(Bool(false))
-  val stickyNonzeroRespReg = RegInit(Bool(false))
+  val busyReg = RegInit(false.B)
+  val stickyBusyReg = RegInit(false.B)
+  val stickyNonzeroRespReg = RegInit(false.B)
 
-  val downgradeOpReg = Reg(init = Bool(false)) // downgrade op because prev. failed.
+  val downgradeOpReg = RegInit(false.B) // downgrade op because prev. failed.
 
   val busy = Wire(Bool())
   val nonzeroResp = Wire(Bool())
 
   val busyResp    = Wire(new DMIAccessCapture(debugAddrBits))
-  val nonbusyResp = Wire(new DMIAccessCapture(debugAddrBits))
```

Comment:
did we just get rid of this because it is unused?

Reply:
yes - it complained about nonbusyResp not being fully initialized. True, since it was never assigned.